src/python/pants/option/option_types.py

```diff
@@ -660,19 +664,52 @@ def _convert_(self, val: Any) -> dict[str, _ValueT]:
 # -----------------------------------------------------------------------------------------------
 # "Specialized" Concrete Option Classes
 # -----------------------------------------------------------------------------------------------
+def _get_tool_name(subsystem_cls: type["Subsystem"]):
+    return getattr(subsystem_cls, "name", subsystem_cls.options_scope)
 
 
-class ArgsListOption(ShellStrListOption):
-    """A homogenous list of shell str options, to be used as arguments passed to some other tool.
+class SkipOption(BoolOption[bool]):
+    """A --skip option (for an invocable tool)."""
+
+    def __new__(cls, goal: str, *other_goals: str):
```

Comment:
Should this take `tool_name` kwarg?

Reply:
It wasn't necessary. The only reason `ArsListOption` takes it is for `./pants run`. However that doesn't make sense to skip :clown_face: